samples/drivers/led_sx1509b_intensity/src/main.c

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <device.h>
+#include <drivers/gpio.h>
+#include <drivers/gpio/gpio_sx1509b.h>
+
+#define GREEN_LED 5
```

Comment:
Please replace the hard-coded values with pins from devicetree, so there's a clue for people who want to adapt this to other boards.

```
-#define GREEN_LED 5
-#define BLUE_LED 6
-#define RED_LED 7
+/* These are for the Nordic Thingy:52.  Other target boards may use
+ * other labels.
+ */
+#define GREEN_LED DT_GPIO_PIN(DT_NODELABEL(led0), gpios)
+#define BLUE_LED DT_GPIO_PIN(DT_NODELABEL(led1), gpios)
+#define RED_LED DT_GPIO_PIN(DT_NODELABEL(led2), gpios)
```
#define GREEN_LED_NODE DT_NODELABEL(led0)+